### PR TITLE
Cleanup: minor improvements to README

### PR DESCRIPTION
**Pull request by nre-ableton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # postgres-pipeline-utils
 
 `postgres-pipeline-utils` is a Jenkins shared pipeline library which makes it easier to
-run commands against Postgres in a Docker container. For information on using this library
-in your Jenkins pipeline scripts, please refer to the [Jenkins shared library
+run commands with Postgres database in a Docker container. For information on using this
```
Comment (rco-ableton) [inline README.md:4]:
```suggestion
run commands with a Postgres database in a Docker container. For information on using this
```

**Review by rco-ableton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nre-ableton) [reply README.md:4]: Whoops, thanks!

**Review by rco-ableton — APPROVED** [review]
LGTM 👍

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**